test/remote/gateways/remote_braintree_blue_test.rb

```diff
@@ -17,6 +17,14 @@ def setup
     }
   end
 
+  def test_supports_network_tokenization_brands
```

Comment:
this is a _remote_ test that is meant to verify we get the correct brands back from braintree